### ISSUE TITLE
[Feature Request] element-ui use very old dependencies(async-validator, deepmerge, throttle-debounce)

### ISSUE THREAD

**Issue by swxEmily** [ElemeFE/element#20180]
### Existing Component
是

### Component Name
element-ui

### Description
Hope to upgrade dependencies' version

**Comment by zazzaz** [comment]
Yes, you can check the dependecies upgrade plan here on our Vue3.0 version.

https://github.com/ElemeFE/element/issues/20061
